### PR TITLE
Parameter mapping: Add option to ignore time-point specific noiseParameters

### PR DESCRIPTION
**Pull request by dweindl**
... in case those are fixed values and noiseFormula consists only of one single parameter.

### CODE REVIEW

File: petab/parameter_mapping.py [hunk 1]
```diff
@@ -42,6 +42,7 @@ def get_optimization_to_simulation_parameter_mapping(
         warn_unmapped: Optional[bool] = True,
         scaled_parameters: bool = False,
         fill_fixed_parameters: bool = True,
+        allow_scalar_timepoint_specific_numeric_noise_parameters: bool = False
```
Comment (yannikschaelte) [inline petab/parameter_mapping.py:45]: this is ugly

**Review by yannikschaelte — APPROVED** [review]
is it nowhere used?

File: petab/parameter_mapping.py (hunk 1 above)
Comment (dweindl) [reply petab/parameter_mapping.py]: :crying_cat_face:

Comment (yannikschaelte) [reply petab/parameter_mapping.py]: ALSCATISPENUMNOIPA!

**Pull request merged**